modules/solid_mechanics/doc/content/source/materials/CappedWeakPlaneStressUpdate.md

```diff
@@ -1 +1,767 @@
+\end{equation}
+
+The tensile flow potential is
+
+\begin{equation}
+g_{1} = p \ ,
+\end{equation}
+
+and the compressive flow potential is
+
+\begin{equation}
+g_{2} = - p \ .
+\end{equation}
+
+The overall flow potential is
+
+\begin{equation}
+g = \left\{
+\begin{array}{ll}
+g_{0} & \mbox{ if } f = f_{0} \\
+g_{1} & \mbox{ if } f = f_{1} \\
+g_{2} & \mbox{ if } f = f_{2}\ .
+\end{array}
+\label{one.g.eqn}
+\right.
+\end{equation}
+
+Obviously there are problems here where $g$ is not defined properly at
+the corners where $f_{0}=f_{1}$ and $f_{0}=f_{2}$ (or even
+$f_{0}=f_{1}=f_{2}$).  This is resolved by using smoothing (more on
+this later).
+
+This plasticity model contains two internal parameters, denote by
+$i_{0}$ and $i_{1}$.   It is assumed
+that
+
+\begin{eqnarray}
+C & = & C(i_{0}) \ , \\
+\phi & = & \phi(i_{0}) \ , \\
+\psi & = & \psi(i_{0}) \ , \\
+S_{T} & = & S_{T}(i_{1}) \ , \\
+S_{C} & = & S_{C}(i_{1}) \ .
+\end{eqnarray}
+
+That is, $i_{0}$ can be thought of as the ``shear'' internal parameter,
+while $i_{1}$ is the ``tensile'' internal parameter.
+
+To complete the definition of this plasticity model, the increments of
+$i_{0}$ and $i_{1}$ during the return-map process must be defined.
+The return-map process involves being provided with a trial stress
+$\sigma^{\mathrm{trial}}$ and an existing value of the internal
+parameters $i^{\mathrm{old}}$, and finding a ``returned'' stress,
+$\sigma$, and internal parameters, $i$, that satisfy
+\begin{eqnarray}
+0 & = & f(\sigma, i) \ . \\\label{f.zero.return}
+\sigma & = & \sigma^{\mathrm{trial}} - E\gamma \frac{\partial
+  g}{\partial\sigma} \ ,
+\end{eqnarray}
+
+where $E$ is the elasticity tensor, and $\gamma$ is a ``plastic
+multiplier'', that must be positive.  The former expresses that the
+stress must be admissible, while the latter is called the ``normality
+condition''.  Loosely speaking, the returned stress lies at a position
+on the yield surface where the normal points to the trial stress
+(actually $E$ and $\partial g/\partial\sigma$ must be used to define
+the ``normal direction'').
+
+Let us express the normality condition in $(p, q)$ space.  The $zz$
+component is easy:
+
+\begin{equation}
+p = \sigma_{zz} = \sigma_{zz}^{\mathrm{trial}} - E_{zzij}\gamma
+\frac{\partial g}{\partial \sigma_{ij}} = \sigma_{zz}^{\mathrm{trial}} - E_{zzzz}\gamma
+\frac{\partial g}{\partial p}  \ ,
+\label{p.norm.eqn}
+\end{equation}
+
+where the last equality holds by assumption (see full list of
+assumptions below).  The $xz$ and $yz$ components are similar:
+
+\begin{equation}
+\sigma_{xz} = \sigma_{xz}^{\mathrm{trial}} - E_{xzxz}\gamma
+\frac{\partial g}{\partial q}\frac{\partial q}{\partial \sigma_{xz}}
+\ . \label{eqn.xz.flow}
+\end{equation}
+
+Another assumption has been made about $E$.  The final term is
+
+\begin{equation}
+\frac{\partial q}{\partial\sigma_{xz}} = \frac{\sigma_{xz}}{q} \ .
+\end{equation}
+
+This means that Eqn~(\ref{eqn.xz.flow}) can be re-written
```

Comment:
There is a different format in .md for referencing equations. I think you can simply use `This means that [eqn.xz.flow]` to properly reference the equation.